src/room/GroupCallView.tsx

```diff
@@ -262,6 +263,7 @@ export function GroupCallView({
           roomIdOrAlias={roomIdOrAlias}
           unencryptedEventsFromUsers={unencryptedEventsFromUsers}
           hideHeader={hideHeader}
+          isOnlyScreenshare={initCallWithoutVideoAndAudio}
```

Comment:
I don't think these are quite the same thing? `initCallWithoutVideoAndAudio` controls whether a call without no audio or video is permitted, but `isOnlyScreenshare` implies we definitely don't have audio or video.

Reply:
`initCallWithoutVideoAndAudio` is a matrix js sdk property describe the behavior of the sdk
`isOnlyScreenshare` is a element call property describe the feature we implementing.

@dbkr What is your suggestion how we can rename one of the property?

Reply:
I think both of these might be the same problem: the `isOnlyScreenshare` and the fact that the buttons are hidden entirely implies that EC won't try to open audio & video devices at all, but the js-sdk will still try to open devices with the flag enabled, the difference is that it will continue if it fails (as I understand it).

I think we probably should choose either:
 * Allowing calls to proceed when they have no audio or video devices, continue to show the buttons (they could just do nothing if no corresponding feed for now rather than retrying `getUserMedia()`): this way the `isOnlyScreenshare` flag is no longer needed since we don't need to change the UI.
 * Have a special mode where calls are *only* screenshares and never try to open audio or video devices and we hide the mute buttons entirely. I feel like this would need a nontrivial amount of UI design though, since you would want to make it a different thing in the UI.

I have a fairly strong preference for the former.

Reply:
@dbkr I will implement this like your first suggestion. But I need this property either, because.. 
In the case the property is true and no track is in the feed i disable the buttons in a call. Even i have to hide the status message `User is connecting...` This view status is covert by the feeds and is not disappearing if the feed has no tracks.

Reply:
I will rename the property in `allowCallWithoutVideoAndAudio`

Reply:
@dbkr: I reduced the `matrix-js-sdk` dependencies to a minimum. Except for one point, the EC works independently of the feature, now.